com.unity.netcode.gameobjects/Runtime/Timing/NetworkTimeSystem.cs

```diff
@@ -82,10 +82,16 @@ public class NetworkTimeSystem
         /// The averaged latency in network ticks between a client and server.
         /// </summary>
         /// <remarks>
-        /// For a distributed authority network topology, this latency is between
-        /// the client and the distributed authority service instance.
+        /// For a distributed authority network topology, this latency is between the client and the
+        /// distributed authority service instance.<br />
+        /// Note: <see cref="Components.NetworkTransform"/> uses this value plus an additional global
+        /// offset <see cref="Components.NetworkTransform.InterpolationBufferTickOffset"/> when interpolation
+        /// is enabled. <br />
+        /// To see the current <see cref="Components.NetworkTransform"/> tick latency: <br />
+        /// - <see cref="Components.NetworkTransform.GetTickLatency"/> <br />
+        /// - <see cref="Components.NetworkTransform.GetTickLatencyInSeconds"/> <br />
         /// </remarks>
-        public int TickLatency = 2;
+        public int TickLatency = 1;
```

Comment:
Will there be any side effects from changing the default of this value?

Reply:
Yes. This was originally created in [PR-3337](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/3337/files#diff-66360097061d73228ae288f38b02f6fed365b52920b5ae021cebb7c5da837e1a) which was not a reasonable value to start with. Tick latency will be adjusted based on the latency from the server (NGO or Rust) as it updates and synchronizes time. However, the host/server should really stay at a tick latency of 1 until we can get clients sending a response back to the server when it gets a network time synchronization so the server can make adjustments dynamically (per client) to this value... until then it should be 1 _(I can explain in better detail in our next 1:1)_